.github/workflows/docker_hub_build.yml

```
@@ -0,0 +1,50 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - 'master'
+      - 'test-branch'
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            tacoma/fqm-execution-service
+          # generate Docker tags based on the following events/attributes
```

```suggestion
          flavor: |
            latest=${{ github.ref == 'refs/heads/master' }}
          # generate Docker tags based on the following events/attributes
```
This should force `master` to be tagged as `latest`.